### DISCUSSION TITLE
2 Olaram Devices?

### DISCUSSION THREAD

**Discussion by cp6969** [rainepretorius/olarm-ha-integration#117] [General]
I already have the Olarm intergration set up and working fine for my home alarm. I would like to intergrate the Olarm for my work alarm. If I try adding a 2nd device I get "unknown error occured" after entering the API key

**Comment by boereworswizard** [comment]
Did you find any workaround?

**Comment by rainepretorius** [comment]
It is not possible to add two Olarms to a single account. You will need to create a separate Olarm account for the work device. Once the second account is set up, generate an API key for that Olarm. You can then integrate the second Olarm using the new API key.

Please note that Olarm enforces a rate limit of one request every 10 seconds per API key, so a single key cannot be used to manage both devices simultaneously.

**Reply by cp6969** [reply]
But I have two seperate Olarm accounts. different locations and different login details. When I try add the 2nd Olarm to HA intergration It just says unknown error

**Reply by boereworswizard** [reply]
> But I have two seperate Olarm accounts. different locations and different login details. When I try add the 2nd Olarm to HA intergration It just says unknown error

The entire integration is broken, you can't setup any new accounts whatsoever.

**Reply by cp6969** [reply]
Oh bother, and I gather this integration is no longer being maintained?
